.githooks/pre-commit

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+LC_ALL=C
+
+local_branch="$(git rev-parse --abbrev-ref HEAD)"
+
+valid_branch_regex="^(feature\/VIH-[0-9]{3})[a-zA-Z0-9._-]+$"
```

Comment:
Hi Valentina, Can we make sure to enforce this format for consistency? We need  Story number then ": " and then a description. Can you amend the regex, readme and the error message to reflect that? Thanks. 

https://tools.hmcts.net/confluence/display/VIH/PR+naming+convention

Reply:
Hi Ru, The regex for branch name I did not change, as branch name could not have ':' or blank in the name. But I have added the new hook 'prepare-commit-msg' which will check if the commit message  prefix VIH-XXXX and if no such prefix then the prefix will be added to commit message by the hook script.

Reply:
Hi, But the requirement is to have origin/feature/VIH-XXX : Description. I can't see this format enforced in the regex. Regex only looks at "VIH-XXX" part.

Reply:
Need to be more like (/feature\/VIH-[0\-9] {3} : /) [a-zA-Z0-9._-]+$

Reply:
The changes done